### DISCUSSION TITLE
Heterogeneous wind direction

### DISCUSSION THREAD

**Discussion by maapasan** [NatLabRockies/floris#803] [Q&A] [answered]
Hello FLORIS team!

I would like to know if it is possible to introduce heterogeneous wind changes in the wind direction (not wind speed).
I think that this can be needed for the study of big wind farms and I remember that it was possible in previous versions of FLORIS.

Thank you very much!!

**Comment by rafmudaf** [comment, marked as answer]
Hi @maapasan no, this is not supported in FLORIS after v2, and it is not currently on our roadmap for upcoming work. However, we can work with you to support adding this functionality, if you're interested to contribute.

**Reply by maapasan** [reply]
Thank you very much for your response @rafmudaf
